lib/billing_facade_client/cost_for_module.rb

```diff
@@ -0,0 +1,42 @@
+module BillingFacadeClient
+  module CostForModule
+    # Returns boolean telling if the response obtained from the billing-facade-mock server is valid
+    # It is considered valid if:
+    # - It does not contain any errors
+    # - The Cost code and Module name in the response match the arguments provided
+    # - The Price obtained is a valid numeric value
+    def validate_response_cost_for_module_name(response, product_name, module_name, cost_code)
+      !!((response) && (!response[:errors]) && 
+        (response[:product] == product_name) &&
+        (response[:cost_code] == cost_code) && (response[:module] == module_name) &&
+        (price_is_valid?(response[:price])))
+    end
+
+    # Given a product name and a module name for it, and a cost code, it performs a query to the billing
+    # facade service and returns the price value
+    def get_cost_information_for_module(product_name, module_name, cost_code)
+      r = connection.post("/price_for_module", msg_request_cost_information_for_module(cost_code, module_name))
```

Comment:
This calls msg_request_cost_information_for_module with only two parameters. `product_name` is missing.